docs/userguide/failover/cluster-failover.md

```diff
@@ -136,6 +136,18 @@ JSONPath expressions follow the Kubernetes specification: <https://kubernetes.io
 
 **Note**: JSONPath expressions by default start searching from the "status" field of the API resource object. For example, to extract "availableReplicas" from a Deployment, the JSONPath expression should be `{.availableReplicas}` instead of `{.status.availableReplicas}`.
 
+#### Configuring Eviction Rate Limiting
```

Comment:
I think that adjusting this section to a third-level heading would be more appropriate.